packages/synapse-react-client/src/lib/assets/icons/Cavatica.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+
+export const Cavatica = () => (
+  <svg
+    width="20"
+    height="20"
+    viewBox="0 0 828 828"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
```

Comment:
Consider using MUI `SvgIcon` (see LockIcon.tsx for an example)

Reply:
Done